js/parse.js

```diff
@@ -376,7 +377,19 @@ function fillCharacterData(characters, numChars, fields) {
         characters[i].bubblesEquipped = [
             mapLookup(largeBubbleMap, charEquippedBubbles[0]),
             mapLookup(largeBubbleMap, charEquippedBubbles[1])
-        ]
+        ];
+
+        const fieldsPrint = JSON.parse(fields.Print.stringValue);
+        const printData = fieldsPrint.slice(5, fieldsPrint.length); // REMOVE 5 '0' ELEMENTS
+        const chunk = 14; // # OF SAMPLE SLOTS OF ONE CHARACTER
+        const relevantPrinterData = printData.slice(i * chunk, (i * chunk + chunk));
+        characters[i].printer = [
+            {
+                item: itemMap[relevantPrinterData[10]],
```

Comment:
Instead of using itemMap directly here (and once more below), use the function mapLoopup(itemMap, data). This makes it really easy to handle not found data